Source/FullScreenPokemon.ts

```diff
@@ -9306,9 +9308,9 @@ module FullScreenPokemon {
         clearSavedData(): void {
             let oldLocalStorage: ItemsHoldr.IItems = this.ItemsHolder.exportItems();
 
-            let collectionKeys: string[] = this.ItemsHolder.getItem(this.StateHolder.getPrefix() + "collectionKeys");
-            for (let collection of collectionKeys) {
-                oldLocalStorage[collection] = this.ItemsHolder.getItem(collection);
+            let collectionKeys: string[] = this.ItemsHolder.getItem("stateCollectionKeys");
+            for (let i: number = 0; collectionKeys && i < collectionKeys.length; i += 1) {
```

Comment:
Whoops, probably accidentally picked the wrong version while fixing merge conflicts.